ironfish/src/network/messages/newTransaction.test.ts

```diff
@@ -1,13 +1,29 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { createNodeTest, useTxSpendsFixture } from '../../testUtilities'
 import { NewTransactionMessage } from './newTransaction'
 
 describe('NewTransaction', () => {
-  it('serializes the object into a buffer and deserializes to the original object', () => {
+  const nodeTest = createNodeTest()
+
+  function expectNewTransactionMessageToMatch(
```

Comment:
I think having a whole separate function here is pretty verbose. We aren't reusing it anywhere and we can just replace with 2 lines. We're going to be deleting this file soon though so not a big deal
```javascript
expect(message.transaction.hash().equals(deserializedMessage.transaction.hash())).toBe(true)
expect({ ...message, transaction: undefined }).toMatchObject({ ...deserializedMessage, transaction: undefined })
```